tmf634/server/src/server.rs

```diff
@@ -86,25 +86,29 @@ pub async fn create(redis_uri: &str, bind: &str, port: &str, https: bool) {
         }
     } else {
         // Using HTTP
-        let server = match hyper::server::Server::try_bind(&addr) {
+        let serve = match hyper::server::Server::try_bind(&addr) {
```

Comment:
Because it was wrong.  There is already a `server` variable for another purpose. This is a [hyper::server::Builder](https://docs.rs/hyper/0.14.6/hyper/server/struct.Builder.html).